src/main/resources/fxml/main.fxml

```diff
@@ -1,22 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <?import javafx.geometry.Insets?>
 <?import javafx.scene.layout.*?>
-<BorderPane prefHeight="800.0" prefWidth="1000.0" xmlns="http://javafx.com/javafx/" xmlns:fx="http://javafx.com/fxml/1">
+
+<BorderPane fx:id="root" prefHeight="800.0" prefWidth="1000.0"
+            xmlns="http://javafx.com/javafx/22.0.1"
+            xmlns:fx="http://javafx.com/fxml/1"
+            fx:controller="org.example.globalgoodsindex.controllers.MainController">
+
     <top>
-        <fx:include source="ui/menu-bar.fxml"/>
+        <fx:include fx:id="menuBarInclude" source="ui/menu-bar.fxml"/>
```

Comment:
U can set `fx;id` for `fx:include` line? Why?

Reply:
this was part of my attempts to sync themesSwitcher between IconThemeSwitcher & MenuThemeSwitcher (will delete it when we will decide definitive we will get rid of Icons :) )